### PR TITLE
chore: update k8s core

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/ui-automation/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,6606 @@
 {
   "name": "trustbloc-ui-automation-test",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (rolsonquadras) [inline test/ui-automation/package-lock.json:4]: Looks like these changes are not related to the PR

**Review by rolsonquadras — APPROVED** [review]

**Review by troyronda — APPROVED** [review]
